app/queries/consumer_apps/find_or_create_by_query.rb

```diff
@@ -28,7 +28,7 @@ def relation
     end
 
     def forem_consumer_app
-      relation.create_or_find_by(app_bundle: ConsumerApp::FOREM_BUNDLE)
+      relation.create_or_find_by(app_bundle: ConsumerApp::FOREM_BUNDLE, team_id: ConsumerApp::FOREM_TEAM_ID)
```

Comment:
Same as above: We want these queries to start creating new instances the Forem Consumer Apps with the `team_id`